site/docs/guide/filter-queries.md

```diff
@@ -201,6 +190,93 @@ bot
 The type inference of `ctx` will scan through the entire call chain and inspect every element of all three `.on` calls.
 As an example, it can detect that `ctx.msg.text` is a required property for the above code snippet.
 
+## Useful Tips
+
+Here are some less known features of filter queries that can come in handy.
+Some of them are a little advanced, so feel free to move on to [the next section](./commands.md).
+
+### Chat Member Updates
+
+You can use the following filter query to receive status updates about your bot.
+
+```ts
+bot.on("my_chat_member"); // start, stop, join, or leave
+```
+
+In private chats, this triggers when the bot is started or stopped.
+In groups, this triggers when the bot is added or removed.
+You can now inspect `ctx.myChatMember` to figure out what exactly happened.
+
+This is not to be confused with
+
+```ts
+bot.on("chat_member");
+```
+
+which can be used to detect status changes of other chat members, such as when people join, get promoted, and so on.
+
+> Note that `chat_member` updates need to be enabled explicitly by specifying `allowed_updates` when starting your bot.
+
+### Combining Queries With Other Methods
+
+You can combine filter queries with other methods on the `Composer` class ([API Reference](https://doc.deno.land/https://deno.land/x/grammy/mod.ts/~/Composer)), such as `command` or `filter`.
+This allows for powerful message handling patterns.
+
+```ts
+bot.on(":forward_date").command("help"); // forwarded /help commands
+
+// Only handle commands in private chats:
+const pm = bot.filter((ctx) => ctx.chat?.type === "private");
+pm.command("start");
+pm.command("help");
+```
+
+### Filtering by Message Sender Type
+
+There are five different possible types of message authors on Telegram.
+
+1. Channel post authors
+2. Automatic forwards from linked channels in discussion groups
+3. Normal user accounts, this includes bots (i.e. “normal” messages)
+4. Admins sending on behalf of the group ([anonymous admins](https://telegram.org/blog/filters-anonymous-admins-comments#anonymous-group-admins))
+5. Users sending messages as one of their channels
+
+You can combine filter queries with other update handling mechanisms to find out the type of the message author.
+
+```ts
+// Channel posts sent by `ctx.senderChat`:
+bot.on("channel_post");
+
+const messages = bot.on("message"); // (only regard messages from here)
+
+// Automatic forward from the channel `ctx.senderChat`:
```

Comment:
```suggestion
// Automatic forward from the channel, `ctx.senderChat`
```

Reply:
The channels name is ctx.senderChat, so it's “the channel ctx.senderChat” such as in “the library grammY”